azure-stack/hci/manage/repair-server.md

```diff
@@ -143,6 +143,9 @@ Follow these steps on another server that is a member of the same Azure Stack HC
     Repair-Server -Name "< Name of the new server>" -LocalAdminCredential $Cred
     ```
 
+    > [!NOTE]
+    > The server name must be the [NetBIOS name](/windows/win32/sysinfo/computer-names) This convention also applies to the cluster name.
+
```

Comment:
Change this to: > The server name must be the [NetBIOS name](/windows/win32/sysinfo/computer-names).